src/utils.ts

```diff
@@ -549,4 +551,23 @@ export function isNumber(num: number): boolean {
 
 export function toVoidP(p: Promise<any>): Promise<void> {
     return p.then(() => { });
+}
+
+export function calculateElapsedTime(startProcessingTime: [number, number]): number {
+    const NanoSecondsPerMillisecond = 1000000;
+    const MillisecondsPerSecond = 1000;
+
+    const ellapsedTime = process.hrtime(startProcessingTime);
+    const ellapsedMilliseconds = ellapsedTime[0] * MillisecondsPerSecond + ellapsedTime[1] / NanoSecondsPerMillisecond;
```

Comment:
isnt this easier to understand a different way => ( `ellapsedTime[0]* 1e9 + ellapsedTime[1])/NanoSecondsPerMillisecond`?
where 1e9 is the number of nanoseconds in a second.

Reply:
Sounds good. Changed.